src/pages/UnlockedWallet/AddressesPage/AddressesTabContent.tsx

```diff
@@ -95,3 +119,48 @@ const ToggleText = styled.div`
   font-weight: var(--fontWeight-semiBold);
   color: ${({ theme }) => theme.font.secondary};
 `
+
+const HeaderMiddle = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 20px;
+  flex: 1;
+`
+
+const TableGrid = styled(Box)`
+  contain: paint; // This is amazing. It replaces "overflow: hidden". Using "overflow" on this prevents us from having a sticky table header.
```

Comment:
This took me many hours to figure out (look at the git history and you'll understand!)

Reply:
Oh wow! The number of times I was struggling with avoid the use of `overflow: hidden` to allow for "stickiness" 🙃.. <3